spec/features/admin_company_management_spec.rb

```diff
@@ -32,6 +32,7 @@ def login
       expect(page).to have_content(companies.first.email)
       expect(page).to have_content(companies.first.city)
       expect(page).to have_content(companies.first.state)
+      expect(page).to have_content(companies.first.status.capitalize)
```

Comment:
yes, it's okay with me too...that's what me and Laura were talking that we will have to change the default value for status and make a new migration if this is not acceptable :)